subprojects/delivery/qapps/src/main/kotlin/com/avito/plugin/QAppsPlugin.kt

```diff
@@ -46,7 +46,7 @@ public class QAppsPlugin : Plugin<Project> {
                     configure(extension, variant)
 
                     if (project.plugins.hasPlugin(SignServicePlugin::class.java)) {
-                        val signTaskProvider = project.tasks.signedApkTaskProvider(variant.name)
+                        val signTaskProvider = project.tasks.legacySignedApkTaskProvider(variant.name)
```

Comment:
Did you think to move this logic to the client side? Or this isn't possible?

Reply:
all "legacy" things will be removed after migration